sktime/tests/_config.py

```diff
@@ -31,9 +31,9 @@
 ONLY_CHANGED_MODULES = False
 
 
-# DO NOT ADD ESTIMATORS HERE ANYMORE
-# ADD TEST SKIPS TO TAG tag tests:skip_all INSTEAD
 EXCLUDE_ESTIMATORS = [
+    # SFA is non-compliant with any transformer interfaces, #2064
+    "SFA",
     # PlateauFinder seems to be broken, see #2259
```

Comment:
these should go back.

you shouldn't delete the warning "DO NOT ADD ESTIMATORS HERE ANYMORE"

and you shouldn't put the "SFA" back since that one was resolved